rcl/test/rcl/test_graph.cpp

```diff
@@ -246,7 +258,7 @@ check_graph_state(
   bool expected_in_tnat,
   size_t number_of_tries)
 {
-  RCUTILS_LOG_INFO_NAMED(ROS_PACKAGE_NAME,
+  RCUTILS_LOG_DEBUG_NAMED(ROS_PACKAGE_NAME,
```

Comment:
I don't think we should necessarily change this in this PR; it muddies the waters a bit (since I'm not 100% sure we want to do this).  The same goes for elsewhere in this test (it's OK for the new code to use DEBUG, since nothing else could be depending on that).

Reply:
Okay, I can change that to info. What do you mean depending on logging?

Reply:
> Okay, I can change that to info. What do you mean depending on logging?

It's possible that there is a higher-level test that looks at the output of this to determine whether this was successful (it doesn't look like that is the case here, but we do have tests that do this).  In any case, these changes are unrelated to this PR, so it is better to split them out.